### PR TITLE
feat: added accessibility value aliases 

### PR DESCRIPTION
**Pull request by dakshbhardwaj**
## Summary
This adds aliasing for accessibility Value, it's used as requested on https://github.com/facebook/react-native/issues/34424.

- [aria-valuemax](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-valuemax) to equivalent [accessibilityValue.max](https://reactnative.dev/docs/accessibility#accessibilityvalue)
- [aria-valuemin](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-valuemin) to equivalent [accessibilityValue.min](https://reactnative.dev/docs/accessibility#accessibilityvalue)
- [aria-valuenow](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-valuenow) to equivalent [accessibilityValue.now](https://reactnative.dev/docs/accessibility#accessibilityvalue)
- [aria-valuetext](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-valuetext) to equivalent [accessibilityValue.text](https://reactnative.dev/docs/accessibility#accessibilityvalue)

## Changelog

[General] [Added] - Add `aria-valuemax`, `aria-valuemin`, `aria-valuenow`, `aria-valuetext` as alias prop to `TouchableOpacity`, `View`, `Pressable` `TouchableHighlight` `TouchableBounce` `TouchableWithoutFeedback` `TouchableOpacity` components

## Test Plan
 - Enable `talkback`.
 - Open the RNTester app and navigate to the Api's tab
 - Go to the `fake Slider Example for Accessibility Value `modes section
 
<Image src="https://user-images.githubusercontent.com/22423684/187472543-05200d8d-2742-4096-a56c-41f81b440a97.png" height="600" width="300" />

### CODE REVIEW

**Comment by jacdebug** [comment]
Thanks for working on this, could you please have a look on failing test.

File: Libraries/Components/Pressable/Pressable.js [hunk 1]
```diff
@@ -215,12 +219,32 @@ function Pressable(props: Props, forwardedRef): React.Node {
...
+        ? props['aria-valuemin']
+        : props.accessibilityValue?.min,
+    now:
+      props['aria-valuenow'] !== null
+        ? props['aria-valuenow']
+        : props.accessibilityValue?.now,
+    text:
+      props['aria-valuetext'] !== null
+        ? props['aria-valuetext']
+        : props.accessibilityValue?.max,
```
Comment (jacdebug) [inline Libraries/Components/Pressable/Pressable.js:238]:
```suggestion
        : props.accessibilityValue?.text,
```

File: Libraries/Components/Touchable/TouchableBounce.js [hunk 2]
```diff
@@ -131,6 +131,25 @@ class TouchableBounce extends React.Component<Props, State> {
...
+          ? this.props['aria-valuemin']
+          : this.props.accessibilityValue?.min,
+      now:
+        this.props['aria-valuenow'] !== null
+          ? this.props['aria-valuenow']
+          : this.props.accessibilityValue?.now,
+      text:
+        this.props['aria-valuetext'] !== null
+          ? this.props['aria-valuetext']
+          : this.props.accessibilityValue?.max,
```
Comment (jacdebug) [inline Libraries/Components/Touchable/TouchableBounce.js:150]:
```suggestion
          : this.props.accessibilityValue?.text,
```

File: Libraries/Components/Touchable/TouchableHighlight.js [hunk 3]
```diff
@@ -291,6 +291,25 @@ class TouchableHighlight extends React.Component<Props, State> {
...
+          ? this.props['aria-valuemin']
+          : this.props.accessibilityValue?.min,
+      now:
+        this.props['aria-valuenow'] !== null
+          ? this.props['aria-valuenow']
+          : this.props.accessibilityValue?.now,
+      text:
+        this.props['aria-valuetext'] !== null
+          ? this.props['aria-valuetext']
+          : this.props.accessibilityValue?.max,
```
Comment (jacdebug) [inline Libraries/Components/Touchable/TouchableHighlight.js:310]:
```suggestion
          : this.props.accessibilityValue?.text,
```

File: Libraries/Components/Touchable/TouchableNativeFeedback.js [hunk 4]
```diff
@@ -259,6 +259,25 @@ class TouchableNativeFeedback extends React.Component<Props, State> {
...
+          ? this.props['aria-valuemin']
+          : this.props.accessibilityValue?.min,
+      now:
+        this.props['aria-valuenow'] !== null
+          ? this.props['aria-valuenow']
+          : this.props.accessibilityValue?.now,
+      text:
+        this.props['aria-valuetext'] !== null
+          ? this.props['aria-valuetext']
+          : this.props.accessibilityValue?.max,
```
Comment (jacdebug) [inline Libraries/Components/Touchable/TouchableNativeFeedback.js:278]:
```suggestion
          : this.props.accessibilityValue?.text,
```

File: Libraries/Components/Touchable/TouchableOpacity.js [hunk 5]
```diff
@@ -220,6 +220,25 @@ class TouchableOpacity extends React.Component<Props, State> {
...
+          ? this.props['aria-valuemin']
+          : this.props.accessibilityValue?.min,
+      now:
+        this.props['aria-valuenow'] !== null
+          ? this.props['aria-valuenow']
+          : this.props.accessibilityValue?.now,
+      text:
+        this.props['aria-valuetext'] !== null
+          ? this.props['aria-valuetext']
+          : this.props.accessibilityValue?.max,
```
Comment (jacdebug) [inline Libraries/Components/Touchable/TouchableOpacity.js:239]:
```suggestion
          : this.props.accessibilityValue?.text,
```

File: packages/rn-tester/js/examples/Accessibility/AccessibilityExample.js [hunk 6]
```diff
@@ -919,6 +919,91 @@ class FakeSliderExample extends React.Component<{}, FakeSliderExampleState> {
...
+                this.increment();
+                break;
+              case 'decrement':
+                this.decrement();
+                break;
+            }
+          }}
+          aria-valuemax={100}
+          aria-valuemin={0}
+          aria-valuenow={this.state.current}>
```
Comment (jacdebug) [inline packages/rn-tester/js/examples/Accessibility/AccessibilityExample.js:971]: Could you please add test for `aria-valuetext` as well

**Review by jacdebug — CHANGES_REQUESTED** [review]
Sorry for late review, please see inline comment for changes requested.

File: Libraries/Components/Pressable/Pressable.js [hunk 7]
```diff
@@ -238,12 +242,32 @@ function Pressable(props: Props, forwardedRef): React.Node {
...
```
Comment (jacdebug) [inline Libraries/Components/Pressable/Pressable.js:261]:
```suggestion
        : props.accessibilityValue?.text,
```

**Review by jacdebug — CHANGES_REQUESTED** [review]
One more left

**Comment by dakshbhardwaj** [comment]
> One more left

@jacdebug pushed the changes

**Comment by jacdebug** [comment]
there are some new conflicts @dakshbhardwaj

**Comment by dakshbhardwaj** [comment]
> there are some new conflicts @dakshbhardwaj

@jacdebug I have resolved the conflicts

File: Libraries/Components/Pressable/Pressable.js [hunk 8]
```diff
@@ -240,6 +244,25 @@ function Pressable(props: Props, forwardedRef): React.Node {
   _accessibilityState =
     disabled != null ? {..._accessibilityState, disabled} : _accessibilityState;
 
+  const accessibilityValue = {
+    max:
+      props['aria-valuemax'] !== null
+        ? props['aria-valuemax']
+        : props.accessibilityValue?.max,
```
Comment (jacdebug) [inline Libraries/Components/Pressable/Pressable.js:251]:
```suggestion
    max: props['aria-valuemax'] ?? props.accessibilityValue?.max,
```

This is more accurate and simple, please update the change for rest of properties on all changed files.

**Review by jacdebug — CHANGES_REQUESTED** [review]
Please see inline commend use nullish coalescing operator is correct and make code more readable.

**Comment by dakshbhardwaj** [comment]
> Please see inline commend use nullish coalescing operator is correct and make code more readable.

@jacdebug I have made the changes

**Review by jacdebug — APPROVED** [review]

**Pull request closed without merging**